src/lib.rs

```diff
@@ -3,17 +3,10 @@
 
 #[cfg(not(feature = "device-selected"))]
 compile_error!(
-    "This crate requires one of the following device features enabled:
-        stm32f301
-        stm32f318
-        stm32f302
-        stm32f303
-        stm32f373
-        stm32f378
-        stm32f334
-        stm32f328
-        stm32f358
-        stm32f398"
+    "This crate requires you to specify your target device as a feautre.  
+    Note that the more specific you pick the device, the more functionality will be available.
+
+    See `Cargo.toml` section 'features' for a comprehensive list of devices (starting with stm32f...)."
```

Comment:
Hm. Not sure if it should stay this way. The list from before is outdated so this change overall is a good idea. But I would'nt point a user from  this create to look at the Cargo.toml  / source code to find out the relevant information. I would rather have the information available through the README or in some kind of way, so the user could find it at docs.rs. 

I know you tackle this in https://github.com/stm32-rs/stm32f3xx-hal/pull/75, so deleting this line for now should be okay. If could stay as well for now, and we could change this in the mentioned PR :)

Reply:
removed it and added a link to #75 in the commit message